nemo_text_processing/text_normalization/en/graph_utils.py

```diff
@@ -79,20 +79,34 @@
 delete_extra_space = pynini.cross(pynini.closure(NEMO_WHITE_SPACE, 1), " ")
 delete_preserve_order = pynini.closure(
     pynutil.delete(" preserve_order: true")
-    | (pynutil.delete(" field_order: \"") + NEMO_NOT_QUOTE + pynutil.delete("\""))
+    | (pynutil.delete(' field_order: "') + NEMO_NOT_QUOTE + pynutil.delete('"'))
 )
 
+# String literal insertions
+insert_username = pynutil.insert('username: "')
```

Comment:
I wouldn't create `insert_double_quote`. Just create a variable named `double_quotes` then call the insert function in the appropriate case. You don't want to limit yourself in case you need a deletion function as well.